MeetingScheduler/Logger/FileLogger.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger
+{
+    public class FileLogger : LogBase
+    {
+        public string FilePath { get; set; }
+
+        public FileLogger()
+        {
+            FilePath = "../../../log.txt";
+        }
+
```

Comment:
Generalno, handlovanje exceptiona svuda fali, a to je pogotovu za logger izuzetno bitno. Narocito kod rukovanja fajlovima. Preporuka je da izvuces sto vise specificnih exceptiona a ne samo genericki.